acra-censor/handlers/whitelist_handler.go

```diff
@@ -28,6 +28,7 @@ func (handler *WhitelistHandler) CheckQuery(query string) (bool, error) {
 	if len(handler.queries) != 0 {
 		//Check that query is in whitelist
 		if !handler.queries[query] {
+			log.WithError(ErrQueryNotInWhitelist).Infof("")
```

Comment:
better to use some useful message to understand logs in a future where it was happened